src/components/AdminSettings/GeneralSettings.vue

```diff
@@ -96,13 +109,23 @@ export default {
 
 	mounted() {
 		this.loading = true
+		this.askGuestUsername = parseInt(loadState('spreed', 'ask_guest_username')) === 1
 		this.conversationsFiles = parseInt(loadState('spreed', 'conversations_files')) === 1
 		this.defaultGroupNotification = defaultGroupNotificationOptions[parseInt(loadState('spreed', 'default_group_notification')) - 1]
 		this.conversationsFilesPublicShares = parseInt(loadState('spreed', 'conversations_files_public_shares')) === 1
 		this.loading = false
 	},
 
 	methods: {
+		saveAskGuestUsername() {
+			this.loadingAskGuestUsername = true
+
+			OCP.AppConfig.setValue('spreed', 'ask_guest_username', this.askGuestUsername ? '1' : '0', {
+				success: function() {
+					this.loadingAskGuestUsername = false
+				}.bind(this),
```

Comment:
```suggestion
				success: () => {
					this.loadingAskGuestUsername = false
				},
```

Reply:
I think it makes sense to keep the uniformity